File: languages/zig/practice/control_flow/if.zig
const std = @import("std");
const print = std.debug.print;

fn main() void {
    var number: i32 = 9;

    if (number < 7) {
        print("{} is less than {}\n", .{ number, 7 });
    } else {
        print("Condition false.\n", .{});
    }

    number = 7;

    if (number != 0) {
        print("Number was something other than zero\n", .{});
    }
}

File: languages/rust/practice/control_flow/if.rs
fn main() {
    let mut number = 9;

    if number < 7 {
        println!("{} is less than {}", number, 7);
    } else {
        println!("Condition false");
    }

    number = 7;

    if number != 0 {
        println!("Number was something other than zero");
    }
}
